packages/cozy-stack-client/src/__tests__/AppCollection.spec.js

```diff
@@ -34,4 +35,58 @@ describe(`AppCollection`, () => {
       expect(resp.data[0]).toHaveDocumentIdentity()
     })
   })
+
+  describe('find', () => {
+    it('should rely internally on DocumentCollection.find()', async () => {
+      jest
+        .spyOn(DocumentCollection.prototype, 'find')
+        .mockImplementation(() => {})
+
+      const collection = new AppCollection(client)
+      await collection.find({ slug: 'test' })
+
+      expect(DocumentCollection.prototype.find).toHaveBeenLastCalledWith({
+        slug: 'test'
+      })
+    })
+  })
```

Comment:
That's implementation detail IMHO. If `DocumentCollection.prototype.find` is well tested, I think we don't need this.